pkg/config/parser_test.go

```diff
@@ -47,17 +47,17 @@ func TestConfigParser(t *testing.T) {
 	if config.Seed != 42 ||
 		config.YAMLSelector != "container" ||
 		config.EndpointPort != 80 ||
-		!strings.HasPrefix(config.TracePath, "data/traces") ||
+		!strings.HasPrefix(config.TracePath, "data/traces/example") ||
 		!strings.HasPrefix(config.OutputPathPrefix, "data/out/experiment") ||
 		config.Granularity != "minute" ||
-		config.IATDistribution != "exponential" ||
-		config.ExperimentDuration != 2 ||
+		config.IATDistribution != "equidistant" ||
```

Comment:
May be we should fixate the configs that are used for tests and move them to separate folder, so we won't need to update these tests that often.

BTW, why do we need to update the default config? Is there something wrong?